test/gtest/cba_find2_infer.cpp

```diff
@@ -162,7 +162,11 @@ TEST_P(GPU_ConvBiasActivFind2InferFusionFind_FP32, ConvBiasActivFind2Float_testF
         {miopenTensorActivationY, out_dev.get()},
         {miopenTensorBias, bias_dev.get()},
     };
-
+    if(solutions.empty())
+    {
+        test_skipped = true;
+        GTEST_SKIP() << "No solutions found!";
+    }
```

Comment:
This is a bad thing, it breaks the whole test. Empty `solutions` should cause the whole test to fail. For now, let's add a device check at the beginning of this function and create a ticket to refactor this test.